src/resources/views/crud/fields/relationship/relationship_select.blade.php

```diff
@@ -14,9 +14,10 @@
     }
 
     // make sure the $field['value'] takes the proper value
-    $current_value = old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? [];
-    
-    if (!empty($current_value) || is_int($current_value)) {
+    $current_value = old_input_value($field['name'], $field['value'] ?? $field['default'] ?? []);
+
+
+    if (!empty($current_value) || is_int($current_value) || is_null($current_value)) {
```

Comment:
Heads-up, changed this to also take into account the `null` value. When can this ever be null? When:
- admin loads up the form
- does NOT fill in this particular field
- validation triggers due to some other field
- then this field will be `null`, not `[]` like it expects
- and no, it will not fall back to `[]` for some reason

Reply:
This should not be needed if our `old_or_value()` helper is working properly. 

When the return from `old()` is `null` the return value should be the field `fallback` - `[]`